spec/rubocop/cop/sorbet/sigils/enforce_sigil_order_spec.rb

```diff
@@ -143,5 +143,26 @@ class Foo; end
           class Foo; end
         RUBY
     end
+
+    it('autocorrects magic comments while removing blank lines') do
+      source = <<~RUBY
+        # frozen_string_literal: true
+
+
+        # typed: true
+
+
+
+        # encoding: utf-8
+        class Foo; end
+      RUBY
+      expect(autocorrect_source(source))
+        .to(eq(<<~RUBY))
+          # encoding: utf-8
+          # typed: true
+          # frozen_string_literal: true
+          class Foo; end
+        RUBY
+    end
```

Comment:
How does it behave with:
```ruby
# frozen_string_literal: true


# typed: true

# encoding: utf-8


class Foo; end
```

Reply:
As expected. I can add a test for it if you want more cases.

Reply:
Yes, that would be great. With these types of manipulations, the first mistake leads to out of bounds access and the removal spills to the surrounding area. So, it would be good to have checks around the affected region too.

Reply:
I was mistaken, it's not as expected in this case. 

It preserves any blank lines after magic comments. I've added the test below to make it clear. Note how blank lines around `class Foo; end` remain same.

Reply:
Yes, that's exactly what I would have expected. Thanks for the test.

Reply:
Thank you for pointing it out. Should've been tested in the first place.